frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewSidePanel/FeatureOverviewSidePanelEnvironmentSwitches/FeatureOverviewSidePanelEnvironmentSwitch/FeatureOverviewSidePanelEnvironmentHider.tsx

```diff
@@ -4,7 +4,7 @@ import { Visibility, VisibilityOff } from '@mui/icons-material';
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
 
 const StyledVisibilityToggle = styled(IconButton, {
-    shouldForwardProp: prop => prop !== 'highlighted',
+    shouldForwardProp: prop => prop !== 'visibilityOff',
```

Comment:
this was generating warnings in console.log. It looks like wrong property was excluded

Reply:
https://github.com/Unleash/unleash/pull/2892 - Seems like you were faster 🙂